plugins/woocommerce-admin/client/marketplace/components/extensions/extensions.tsx

```diff
@@ -44,38 +44,8 @@ export default function Extensions(): JSX.Element {
 			params.append( 'country', wccomSettings.storeCountry );
 		}
 
-		const wccomSearchEndpoint =
-			MARKETPLACE_HOST +
-			MARKETPLACE_SEARCH_API_PATH +
-			'?' +
-			params.toString();
-
-		// Fetch data from WCCOM API
-		fetch( wccomSearchEndpoint )
-			.then( ( response ) => response.json() )
-			.then( ( response ) => {
-				/**
-				 * Product card component expects a Product type.
-				 * So we build that object from the API response.
-				 */
-				const products = response.products.map(
-					( product: SearchAPIProductType ): Product => {
-						return {
-							id: product.id,
-							title: product.title,
-							description: product.excerpt,
-							vendorName: product.vendor_name,
-							vendorUrl: product.vendor_url,
-							icon: product.icon,
-							url: product.link,
-							// Due to backwards compatibility, raw_price is from search API, price is from featured API
-							price: product.raw_price ?? product.price,
-							averageRating: product.rating ?? 0,
-							reviewsCount: product.reviews_count ?? 0,
-						};
-					}
-				);
-
```

Comment:
I've moved this code into `functions.tsx` alongside the other search API-hitting functions, where it probably belongs, as part of this change.